cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+# Portfolio
+
+## Cloud9
+
+- Go to https://c9.io, click the `Sign Up` button, and create an account
+- Click `Go to your dashboard`
```

Comment:
with A/B testing on cloud9 screens may look different than in your tutorial. Also, the first time I did this part I used my Cloud 9 account I already used before so i never had the welcome screen and had to try to find the settings to get to changing soft tabs, enable auto-save and couldn't really find minimal editor.